dao/src/integ-test/resources/uk/gov/moj/sdt/dao/sql/PurgeDaoTest.sql

```diff
@@ -34,6 +42,9 @@ INSERT INTO bulk_submissions(
 INSERT INTO bulk_submissions(
   bulk_submission_id, bulk_customer_id, target_application_id, service_request_id, sdt_bulk_reference, customer_reference, created_date, number_of_requests, bulk_submission_status, completed_date, updated_date, error_code, error_text, version_number, bulk_payload)
   VALUES (17, 1, 4, 9, 'BR107', 'CR108', (now() - INTERVAL '23 day'), 1, 'BSSTATUS1', now(), now(), 'EC107', 'This is error 7', 1, 'BLK PAYLOAD 4');
+INSERT INTO bulk_submissions(
+bulk_submission_id, bulk_customer_id, target_application_id, service_request_id, sdt_bulk_reference, customer_reference, created_date, number_of_requests, bulk_submission_status, completed_date, updated_date, error_code, error_text, version_number, bulk_payload)
+VALUES (18, 1, 4, 9, 'BR107', 'CR108', (now() - INTERVAL '23 day'), 1, 'BSSTATUS1', now(), now(), 'EC107', 'This is error 7', 1, 'BLK PAYLOAD 4');
```

Comment:
Out of interest, what's this new test data needed for?

Reply:
changed :

INSERT INTO error_logs(
error_log_id, individual_request_id, error_code, created_date, updated_date, version_number, error_text)
VALUES (1006, 109, 'CODE4', (now() - INTERVAL '20 day'), now(), 1, 'Error number 3');

To

INSERT INTO error_logs(
error_log_id, individual_request_id, error_code, created_date, updated_date, version_number, error_text)
VALUES (1006, 112, 'CODE4', (now() - INTERVAL '20 day'), now(), 1, 'Error number 3');

you know this is issue you are getting when you run the Drurun (because of the association we have added as part of JPA migration)